lua/lvim/core/cmp.lua

```diff
@@ -138,7 +139,7 @@ M.config = function()
       return lvim.builtin.cmp.active
     end,
     confirm_opts = {
-      behavior = cmp.ConfirmBehavior.Replace,
+      behavior = cmp_types.ConfirmBehavior.Replace,
```

Comment:
this is still in `cmp`, see https://github.com/hrsh7th/nvim-cmp/blob/e7e2ef7031db8e0cfad7ad9916ec766ddd9dbb3b/lua/cmp/init.lua#L188

```suggestion
      behavior = cmp.ConfirmBehavior.Replace,
```